### DISCUSSION TITLE
ActivityPub (Mastodon) blocks

### DISCUSSION THREAD

**Discussion by erlend-sh** [blockprotocol/blockprotocol#1236] [Ideas]
For starters: Whatever you’ve already got for Twitter.

But with the openness of the AP protocol, I think the potential for further novelty is huge.

**Comment by jhulten** [comment]
I strongly agree. Decoupling the presentation of a type from the service providing it (photos in Pixelfed, short form in Masto, discussion boards in Lemmy) would be a huge gain.
